platform.h

```diff
@@ -16,8 +16,20 @@ enum log_priority {
 
 // override these in platform.cpp
 namespace platform {
+/// Whether the platform has hardware KEY2 support.
+/// If false, flashcart_core will do KEY2 in software. (TODO IMPL)
+/// If unset, defaults to false.
 extern const bool HAS_HW_KEY2;
 
+/// Whether the platform can reset a cart, whether electronically or by asking
+/// the user to physically reinsert the card.
+/// If unset, defaults to false.
+extern const bool CAN_RESET;
```

Comment:
most platform can support reset, just one platform can't.
how about default value change to `true`?

Reply:
Better to be conservative by default.

I'd rather just remove the weak symbol, so platforms _need_ to define these.

Reply:
I like it.